packages/components/Tabs/src/Tabs.tsx

```diff
@@ -82,10 +90,43 @@ export const Tabs = compose<TabsType>({
 
     const styleProps = useStyling(userProps, (override: string) => state[override] || userProps[override]);
 
+
+    if (Platform.OS == 'windows') {
+      const onKeyDown = (ev: any) => {
+        if (ev.nativeEvent.key === 'ArrowRight' || ev.nativeEvent.key === 'ArrowLeft') {
+          const length = state.context.enabledKeys.length;
+          const currTabItemIndex = state.context.enabledKeys.findIndex(x => x == state.context.selectedKey)
+          let newCurrTabItemIndex;
+          if (ev.nativeEvent.key === 'ArrowRight') {
+            if (isCircularNavigation || !(currTabItemIndex + 1 == length)) {
+              newCurrTabItemIndex = (currTabItemIndex + 1) % length;
+              state.context.selectedKey = state.context.enabledKeys[newCurrTabItemIndex];
+              data.onKeySelect(state.context.selectedKey);
+            }
+          }
+          if (ev.nativeEvent.key === 'ArrowLeft') {
+            if (isCircularNavigation || !(currTabItemIndex == 0)) {
+              newCurrTabItemIndex = (currTabItemIndex - 1 + length) % length;
+              state.context.selectedKey = state.context.enabledKeys[newCurrTabItemIndex];
+              data.onKeySelect(state.context.selectedKey);
+            }
+          }
+        }
+      };
+
+      const slotProps = mergeSettings<TabsSlotProps>(styleProps, {
```

Comment:
nit, I would still name this `windowsSlotProps` or something similar for clarity, along with a comment on how we don't have FocusZone there so we need the extra focusZoneRef / onKeyDown callback, along with maybe a bug tracking bringing FocusZone to UWP windows.

Reply:
I tried renaming it to windowsSlotProps, but the tester app crashes if that variable is not called slotProps. I will add the comment on the props with a bug.

Reply:
oh, interesting. I didn't know about that thanks!